typescript/test/utils/mock-bridge.ts

```diff
@@ -314,4 +326,40 @@ export class MockBridge implements Bridge {
   walletRegistry(): Promise<WalletRegistry> {
     throw new Error("not implemented")
   }
+
+  buildRedemptionData(
```

Comment:
Where is it used? 🤔

Reply:
It's a left over- no longer needed because we've refactored the `reuqestRedemption` fn in 9249ee5 and 06601b9. Removed in e814ecaeeb083f4d66a2882007de4b0b7320e4b4

Reply:
What about `buildRedemptionDataLog` and `BuildRedemptionDataLogEntry`?

Reply:
> What about `buildRedemptionDataLog` and `BuildRedemptionDataLogEntry`?

🦅 👁️ thanks for catching this! See 5f0365d.